service/matching/config.go

```diff
@@ -113,6 +115,7 @@ func NewConfig(dc *dynamicconfig.Collection) *Config {
 		ThrottledLogRPS:                 dc.GetIntProperty(dynamicconfig.MatchingThrottledLogRPS, 20),
 		NumTaskqueueWritePartitions:     dc.GetIntPropertyFilteredByTaskQueueInfo(dynamicconfig.MatchingNumTaskqueueWritePartitions, dynamicconfig.DefaultNumTaskQueuePartitions),
 		NumTaskqueueReadPartitions:      dc.GetIntPropertyFilteredByTaskQueueInfo(dynamicconfig.MatchingNumTaskqueueReadPartitions, dynamicconfig.DefaultNumTaskQueuePartitions),
+		TaskqueuePartitionDispatchRate:  dc.GetFloatPropertyFilteredByTaskQueueInfo(dynamicconfig.MatchingTaskqueuePartitionDispatchRate, 2000),
```

Comment:
fyi, default out-of-box is 4, which makes it 500 per partition. just want to make sure this won't cause any unexpected behavior if someone upgrades

Reply:
500 per partition should be fairly large